react/features/participants-pane/components/breakout-rooms/components/native/CollapsibleList.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+
+import React, { useCallback, useState } from 'react';
+import { Text, TouchableOpacity, View } from 'react-native';
+
+import { Icon, IconArrowDown, IconArrowUp } from '../../../../../base/icons';
+import { StyleType } from '../../../../../base/styles';
+
+import styles from './styles';
+
+type Props = {
+
+    /**
+     * The children to be displayed within this list.
+     */
+    children: React$Node,
+
+    /**
+     * Additional style to be appended to the CollapsibleList container.
+     */
+    containerStyle?: StyleType,
+
+    /**
+     * Callback to invoke when the {@code CollapsibleList} is long pressed.
+     */
+    onLongPress?: Function,
+
+    /**
+     * Collapsable list title.
+     */
+    title: Object
+}
+
+export const CollapsibleList
+    = ({ children, containerStyle, onLongPress, title }: Props) => {
+        const [ collapsed, setCollapsed ] = useState(false);
+        const _toggleCollapsed = useCallback(() => {
+            setCollapsed(!collapsed);
+        }, [ collapsed ]);
+
+        return (
+            <View style = { collapsed && containerStyle }>
+                <TouchableOpacity
+                    onLongPress = { onLongPress }
+                    onPress = { _toggleCollapsed }
+                    style = { styles.collapsibleList }>
+                    <TouchableOpacity
+                        onPress = { _toggleCollapsed }
+                        style = { styles.arrowIcon }>
+                        <Icon
+                            size = { 18 }
+                            src = { collapsed ? IconArrowUp : IconArrowDown } />
+                    </TouchableOpacity>
+                    <Text style = { styles.listTile }>
+                        {
+                            title
+                        }
+                    </Text>
+                </TouchableOpacity>
+                {
+                    collapsed && children
```

Comment:
Shouldn't this check be nagated?

Reply:
We should show the content if the arrow is up aka is the state is collapsed.